.gitignore

```diff
@@ -98,6 +98,12 @@ target
 !.vscode/extensions.json
 *.code-workspace
 
+### IntelliJ ###
+.idea/
+
+### Vim ###
+*.swp
+
```

Comment:
I think we should rather have IDE-specific ignore patterns in a user-global .gitignore, rather than in each project?

(If we add them here, it creates a messy proliferation of N×M combinations of projects and IDEs, which obscures what the project itself intentionally ignores.)